src/popup/router/pages/Send.vue

```diff
@@ -200,27 +201,46 @@ export default {
     },
     async fetchFee() {
       await this.$watchUntilTruly(() => this.sdk);
-      this.fee = await calculateFee(
-        this.token === 0 ? TX_TYPES.txSign : TX_TYPES.contractCall,
-        this.sdk.Ae.defaults,
-      );
+      this.fee = await calculateFee(!this.selectedToken ? TX_TYPES.txSign : TX_TYPES.contractCall, {
```

Comment:
```suggestion
      this.fee = calculateFee(!this.selectedToken ? TX_TYPES.txSign : TX_TYPES.contractCall, {
```
`calculateFee` is a sync function